test/contrib/rails/database_test.rb

```diff
@@ -37,6 +37,31 @@ class DatabaseTracingTest < ActiveSupport::TestCase
     assert_nil(span.get_tag('sql.query'))
   end
 
+  test 'active record traces instantiation' do
+    # Only supported in Rails 4.2+
+    if Rails.version >= '4.2'
+      begin
+        Article.create(title: 'Instantiation test')
+        @tracer.writer.spans # Clear spans
+
+        # make the query and assert the proper spans
+        Article.all.entries
+        spans = @tracer.writer.spans
+        assert_equal(2, spans.length)
+
+        instantiation_span = spans.first
+        assert_equal(instantiation_span.name, 'active_record.instantiation')
+        assert_equal(instantiation_span.span_type, 'ruby')
```

Comment:
The `span_type` should not be `ruby` since it's not recognized by our backend. Probably it should be `db` since it's part of a database transaction? `web` applies as well, unless you're using AR standalone (that could be just a script).

Reply:
Ahh, didn't know we can't specify our own types here, which is why I gave it the name I thought more suitable.

Does either `web` or `db` apply? I think the answer would be no: instantiating objects is not a `web` kind of action, nor is it strictly a `db` thing either. Although it's probably closer to `db` than to web in the sense that its for the purpose of the DB, I'd think this could cause us some headaches in the future if, for the purposes of doing statistics or other useful product implementation, we were trying to compare spans of type `db` to one other, and one isn't actually a DB query at all.

This might be a little out of scope of what this PR is for, but I'd suggest maybe we should add a `ruby` or custom span type, because there are many spans that aren't `web` or `db`. This came up once already while trying to implement GraphQL (which is also neither of those things), so I think this is adding to a compelling case to examine what span types we have at our disposal.

Reply:
Okay.